java/domain/src/main/java/com/demo/domain/util/ShortUrlUtil.java

```diff
@@ -0,0 +1,179 @@
+package com.demo.domain.util;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.security.MessageDigest;
+import java.util.Map;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @Author fanzj
+ * @Date 2022/4/2 11:25
+ * @Version 3.0
+ * @Description
+ */
+public class ShortUrlUtil {
+
+    private static HazelcastInstance hazelcastInstance = SpringBeanUtil.getBean("hazelcastInstance", HazelcastInstance.class);
+
+    private static Logger LOGGER = LoggerFactory.getLogger(ShortUrlUtil.class);
+
+    /**
+     * 域名地址-长短映射: 增加需求设置有效期，有效期可以存到v，或另开队列
+     **/
+    //private static Map<String, String> DOMAIN_URL_MAP = new LRUCache<>(MAX_CAPACITY);
+
+    public static IMap<String, String> DOMAIN_URL_MAP = hazelcastInstance.getMap(Constant.MAP_NAME);
+
+    /**
+     * 加密混合的key
+     **/
+    private static final String KEY = "1qaz2wsx3edc";
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 183be6ea-1061-41e9-be1f-c1d830c7c918 -->